scania-digital-design-system/sdds
# Add an option for a smaller min-width on textfield

**Describe pull-request**  
Adds an option for a smaller `min-width` on the textfield component. The default is un-touched.

**Solving issue**  
Fixes: #278 (and perhaps also [AB#1381](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1381)).

**How to test**  
Create a textfield, and supply `width='small'` to get a textfield that has a `min-width` of `104px`
`width='default'` or doing nothing should set `min-width` to `208px`.

**Screenshots**  
![bild](https://user-images.githubusercontent.com/89378987/156754800-b033c66d-ced0-4ada-af44-ec2332920456.png)
`width='small'` to the left and the default to the right.

**Additional context**  
I'm not sure on how this project prefers to handle these kinds of customizations, but I guess this can serve as a template at least.

(tested in Storybook; `min-width: 104px` is somewhat respected, but other properties seems to set the `width` to `180.5px`
